blackbox/stratis-blackbox-run.sh

```diff
@@ -80,6 +80,15 @@ git clone $STRATISD_REPO
 echo "Most recent commits for ${STRATISD_N}:"
 ( cd $STRATISD_N; git status; git log --format="%h %ci :: %s" | head -20 )
 mv -v ${STRATISD_N} ${STRATISD_N}-${STRATISD_V}
+
+# If the vendor directory exists, remove it and recreate it.
```

Comment:
Can you just say remove it in here and the message, not recreate, since cargo will do the recreating later?

Reply:
Resolved.